pelican/settings.py

```diff
@@ -245,6 +239,139 @@ def get_jinja_environment(settings):
     return settings
 
 
+def handle_deprecated_settings(settings):
+    """Converts deprecated settings and issues warnings. Issues an exception
+    if both old and new setting is specified.
+    """
+    settings = copy.deepcopy(settings)
```

Comment:
Why copy?

Reply:
So the original dictionary isn't affected. Mostly out of principle.

Reply:
Understandable, but meh... `configure_settings` does similar stuff without copy :). It's just unnecessary. We don't really care what happens to the original.